### DISCUSSION TITLE
TYPO in a dependency fixed version

### DISCUSSION THREAD

**Discussion by Mati-Q-A** [aquasecurity/trivy#8146] [False Detection]
### Description

i'm using **ch.qos.logback:logback-core** in a project, and trivy detected a vulnerabilty, that's right, but the fixed version in the database is wrong, the final fixed version in maven central is 1.5.13 (https://mvnrepository.com/artifact/ch.qos.logback/logback-core/1.5.13) but trivy is looking for 1.15.13.
 
[Pipeline log](https://github.com/user-attachments/assets/b1116cb7-c768-4263-9439-2ce2237b4a4c)

### Desired Behavior

Trivy must look for 1.5.13 fixed version

### Actual Behavior

Trivy is looking for 1.15.13 fixed version

### Reproduction Steps

```bash
1.
2.
3.
...
```

### Target

Container Image

### Scanner

Vulnerability

### Output Format

Table

### Mode

Client/Server

### Debug Output

```bash
-
```

### Operating System

linux

### Version

```bash
0.55.2-1
```

### Checklist

- [ ] Run `trivy clean --all`
- [ ] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by angelmunozh** [comment]
The correct library is "1.5.13"
https://secalerts.co/vulnerability/GHSA-6v67-2wr5-gvf4

**Comment by DmitriyLewen** [comment]
Hello @Mati-Q-A @angelmunozh 
GitHub database contains typo.
I already fixed - https://github.com/github/advisory-database/commit/d7926d4e0e0953391d4f6be22e75551a98de82d2#diff-000306eca21b602c732b2ef7fcf8981feba8463c01767ca94dfba763a0f080af
```bash
➜ trivy -q rootfs ./logback-core-1.5.12.jar 

Java (jar)

Total: 2 (UNKNOWN: 0, LOW: 1, MEDIUM: 1, HIGH: 0, CRITICAL: 0)

┌───────────────────────────────────────────────────────┬────────────────┬──────────┬────────┬───────────────────┬───────────────┬──────────────────────────────────────────────────────────┐
│                        Library                        │ Vulnerability  │ Severity │ Status │ Installed Version │ Fixed Version │                          Title                           │
├───────────────────────────────────────────────────────┼────────────────┼──────────┼────────┼───────────────────┼───────────────┼──────────────────────────────────────────────────────────┤
│ ch.qos.logback:logback-core (logback-core-1.5.12.jar) │ CVE-2024-12798 │ MEDIUM   │ fixed  │ 1.5.12            │ 1.5.13        │ logback-core: arbitrary code execution via               │
│                                                       │                │          │        │                   │               │ JaninoEventEvaluator                                     │
│                                                       │                │          │        │                   │               │ https://avd.aquasec.com/nvd/cve-2024-12798               │
│                                                       ├────────────────┼──────────┤        │                   │               ├──────────────────────────────────────────────────────────┤
│                                                       │ CVE-2024-12801 │ LOW      │        │                   │               │ logback-core: SaxEventRecorder vulnerable to Server-Side │
│                                                       │                │          │        │                   │               │ Request Forgery (SSRF) attacks                           │
│                                                       │                │          │        │                   │               │ https://avd.aquasec.com/nvd/cve-2024-12801               │
└───────────────────────────────────────────────────────┴────────────────┴──────────┴────────┴───────────────────┴───────────────┴──────────────────────────────────────────────────────────┘
```

Regards, Dmitriy

**Reply by Mati-Q-A** [reply]
Thanks a lot @DmitriyLewen

**Reply by knqyf263** [reply]
@angelmunozh @Mati-Q-A It would be appreciated if you follow [this guideline](https://trivy.dev/latest/community/contribute/discussion/#false-detection) next time when you find something wrong in advisories.

**Reply by Mati-Q-A** [reply]
Okey @knqyf263 first time, sorry!
